src/components/PostPageDetail/elements/Content.js

```diff
@@ -318,7 +323,10 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.transparent
   },
   contensStyle: (paddingBottom) => ({
-    paddingBottom
+    paddingBottom,
+    borderRightWidth: 1,
+    borderLeftWidth: 1,
+    borderColor: COLORS.darkGray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of border styling to `contensStyle` is a good change as it enhances the visual appearance of the component. However, please ensure that these changes do not affect other components that use `contensStyle`.

```diff
- paddingBottom
+ paddingBottom,
+ borderRightWidth: 1,
+ borderLeftWidth: 1,
+ borderColor: COLORS.darkGray
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->